### ISSUE TITLE
Space Laster

### ISSUE THREAD

**Issue by dakota-osu** [cs361w2019-osu/sprint1-project-group-3#66]
As a player, i would like a space laser weapon as to be able to deal with enemy submarines

"The player receives the activation codes for the space laser only after sinking the first enemy ship and replaces the conventional bomb in the player's arsenal"

**Comment by strozzi1** [comment]
Branch feature/laser merged into master!
